src/utils/fmt_logging.h

```diff
@@ -107,6 +162,11 @@
             var1 < var2, _v1 < _v2, "{} vs {} {}", _v1, _v2, fmt::format(__VA_ARGS__));            \
     } while (false)
 
+#define CHECK_STREQ(var1, var2) CHECK_STREQ_MSG(var1, var2, "")
+#define CHECK_STRNE(var1, var2) CHECK_STRNE_MSG(var1, var2, "")
+#define CHECK_STRCASEEQ(var1, var2) CHECK_STRCASEEQ_MSG(var1, var2, "")
```

Comment:
According to the macro names, it's confused to judge which one is case sensitive.
I found there is no caller currently, can you remove `CHECK_STRCASEEQ/NE`?

Reply:
Actually the naming convention has followed `ASSERT_STREQ/NE` and `ASSERT_STRCASEEQ/NE` of google test. And now `CHECK_STRCASEEQ/NE` is not used in fact, I can remove them.